mobile/build.gradle

```diff
@@ -20,37 +21,39 @@ repositories {
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "22.0.1"
-
+    buildToolsVersion '25'
     defaultConfig {
         applicationId "com.dexdrip.stephenblack.nightwatch"
-        minSdkVersion 15
+        minSdkVersion 20
```

Comment:
I can change it to 19 and test it tonight.  I thought I had pulled it far enough back.

I don't have any kitkat devices so I can't test it there. Is it enough that it compiles and runs?